### PR TITLE
Add assets/ dir to cipublish script

### PR DESCRIPTION
**Pull request by kellyi**
## Overview

This PR adds the `assets/` dir to the cipublish script so that Jenkins will include the fonts and illustration from #27 in release builds.

Connects #28 
Connects #25 

## Testing

* get this branch, then run `cipublish`
* move that zip file to its own directory and unzip it
* serve the app and verify that you see the illustration on the "Sensors" spoke and that the new fonts and layout are visible on the other spokes

Once this is in I'll make another release so that Jenkins will have all the changes in the zip.

### CODE REVIEW

**Comment by rajadain** [comment]
Taking a look now.

File: scripts/cipublish [hunk 1]
```diff
@@ -16,7 +16,7 @@ if [ "${BASH_SOURCE}[0]}" ]; then
 
     git rev-parse HEAD > version.txt
     zip -r dist/pwd-river-dispatches-release.zip usgs-details.html version.txt \
-        images/ CHANGELOG.md media/ edge_includes/ Dispatches_v04t* -x *.zip
-
+        assets/ images/ CHANGELOG.md media/ edge_includes/ Dispatches_v04t* -x \
```
Comment (rajadain) [inline scripts/cipublish]: Prefer keeping `-x *.zip` on the same line, as they are a key-value pair.

**Review by rajadain — APPROVED** [review]
+1 tested. The assets were included in the zip, and showed up correctly when run from the unzipped distribution.

Because of the many page reloads, in some cases I found the font to never load and the value to never show, but that may be because I was using `python -m SimpleHTTPServer` to run the distribution rather than the `express` server for development. Which is to say that when using a proper server, the app works correctly.

File: scripts/cipublish (hunk 1 above)
Comment (kellyi) [reply scripts/cipublish]: Sounds good to me. I'll move `-x` to the ensuing line.

**Comment by kellyi** [comment]
Thanks for taking a look at this! Going to merge.

**Pull request merged**